### PR TITLE
minor code cleanup.

### PR DESCRIPTION
**Pull request by escalonn**
adds:
- stuff

fixes:
- other stuff

progress:
- [x] task 1

### CODE REVIEW

**Comment by escalonn** [comment]
looks good

File: StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs [hunk 1]
```diff
@@ -36,18 +39,19 @@ public async Task<IEnumerable<BusinessLogic.Objects.Customer>> GetAllCustomersAs
                     zip = e.Zip
 
                 }
-            }).ToHashSet() ;
+            }).ToHashSet();
         }
-        public async Task AddCustomerAsync(BusinessLogic.Objects.Customer BLCustomer)
+
+        public async Task AddCustomerAsync(BusinessLogic.Objects.Customer blCustomer)
```
Comment (escalonn) [inline StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs:45]: let's add xml docs to this method while we're at it

File: StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs [hunk 2]
```diff
@@ -87,7 +91,7 @@ public async Task<BusinessLogic.Objects.Store> GetStoreInformation(int StoreID)
             }
             catch (Exception e)
             {
-                throw new Exception("Something went wrong in getting the store's information: " + e.Message);
+                throw new Exception("Something went wrong in getting the store's information", e);
```
Comment (escalonn) [inline StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs:94]: probably throw InvalidOperationException instead

File: StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs [hunk 3]
```diff
@@ -113,8 +117,9 @@ public async Task<List<Order>> GetListAllOrdersForStore(int storeID)
                 }
                 return BLListOrders;
             }
-            catch
+            catch (Exception e)
             {
+                _logger.LogError(e, "Exception during retrieving order information for store {storeId}", storeID);
                 throw new Exception("Failed to retrieve order information for store number: " + storeID);
```
Comment (escalonn) [inline StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs:123]: include innerException

**Review by escalonn — COMMENTED** [review]
looks good

File: StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs (hunk 1 above)
Comment (kmanrulze) [reply StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs:45]: aH DARN

File: StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs (hunk 2 above)
Comment (kmanrulze) [reply StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs:94]: WHY Tho

File: StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs (hunk 3 above)
Comment (kmanrulze) [reply StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs:123]: Im gonna include CasualFridayException instead 😎

**Pull request closed without merging**